tests/ExecutorTest.php

```diff
@@ -190,40 +188,47 @@ public function provideScenarios(): array
             [
                 'image' => 'openruntimes/node:v2-18.0',
                 'entrypoint' => 'index.js',
-                'folder' => 'node-empty-object',
+                'folder' => 'node-v2',
+                'version' => 'v2',
                 'assertions' => function ($response) {
-                    $this->assertEquals('completed', $response['body']['status']);
-                    $this->assertEquals('{}', $response['body']['response']);
-                    $this->assertEmpty($response['body']['stdout']);
-                    $this->assertEmpty($response['body']['stderr']);
+                    $this->assertEquals(200, $response['headers']['status-code']);
+                    $this->assertEquals(200, $response['body']['statusCode']);
+                    $this->assertEquals('{"message":"Hello Open Runtimes 👋"}', $response['body']['body']);
+                    $this->assertEmpty($response['body']['logs']);
+                    $this->assertEmpty($response['body']['errors']);
                 }
             ],
             [
-                'image' => 'openruntimes/node:v2-18.0',
+                'image' => 'openruntimes/node:v3-18.0',
```

Comment:
Now that we have both v3 and v2 images, does the warm up task in the beginning also pull the v3 images?

Reply:
Only v3 images. v2 are no longer pulled.